src/pages/settings/settings.tsx

```diff
@@ -1,9 +1,188 @@
-import { TileInfo } from "@govtechsg/tradetrust-ui-components";
-import React, { FunctionComponent } from "react";
+import { TileInfo, TileInfoProps } from "@govtechsg/tradetrust-ui-components";
+import React, { FunctionComponent, SVGProps } from "react";
 import { Link } from "react-router-dom";
 import { Helmet } from "react-helmet";
 import { Page } from "../../components/Layout/Page";
 
+interface Settings extends TileInfoProps {
+  pathLink: string;
+}
+
+const addressBookIcon: React.FunctionComponent<SVGProps<SVGElement>> = () => {
+  return (
+    <svg width="52" height="59" viewBox="0 0 52 59" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
shift custom icons to common ui, then name maybe `IconAddressBook`. same goes with `IconResolverAddress`